defaults/main.yml

```diff
@@ -3,3 +3,8 @@ list_sa:
 
 list_of_dict_sa_with_roles:
   - { sa_name: "example_sa", role: 'view' }
+
+allowed_domains:
+  - example-devel.com/foo
```

Comment:
I'm not sure if it's safe to use such defaults in role, it should stay empty probably